contracts/src/v0.8/automation/dev/v2_3/AutomationRegistrar2_3.sol

```diff
@@ -120,19 +120,7 @@ contract AutomationRegistrar2_3 is TypeAndVersionInterface, ConfirmedOwner, IERC
   mapping(bytes32 => PendingRequest) private s_pendingRequests;
   mapping(uint8 => TriggerRegistrationStorage) private s_triggerRegistrations;
 
-  event RegistrationRequested(
-    bytes32 indexed hash,
-    string name,
-    bytes encryptedEmail,
-    address indexed upkeepContract,
```

Comment:
Hey @RyanRHall / @FelixFan1992 : this refactoring results in a feature change. This upkeepContract is indexed and address is recommended to be indexed, but when we wrap the params with the RegistrationParams struct, we lose this indexed address.

Not a strong opinion but I would recommend keeping the original structure to have the contract indexed and this old structure can also make the atlas changes easier/cleaner (have to replicate the struct in test).